gtfs/gtfs_utils/gtfs_utils/configuration.py

```diff
@@ -69,6 +69,7 @@ class Configuration:
     use_data_from_today: bool = True
     date_range: List[str] = None
     future_days_count: int = 0
+    max_gtfs_size_in_bytes: numbers.Number = float('inf')
```

Comment:
Since I wanted to be able to use int in the config.json, 
and `dict_to_dataclass` raised exception since `int` values are not instance of `float`

Reply:
OK. Can't you just use `int` in all cases?

Reply:
I'll change that to int

Reply:
I tried changing that to int, and use None as default value in case the parameter is not defined in the configuration. That caused and caused the same problem since `None` is not instance of `int`.
So if we want to be able to not set the parameter in the config file, than the default value must be an instance of the hinted type (I think changing the config parser would be bad idea). 
We could use `sys.maxsize` as int biggest value, or just a really big number but both seems less clear than the `numbers.Number` option.

what do you think?

Reply:
I prefer `sys.maxsize`, since if you expect to get an integer, the type hint should be `int`.